src/FormFields/SessionManagerField.php

```diff
@@ -118,7 +119,7 @@ public function getSchemaDataDefaults()
      */
     protected function getLoginSessions(Member $member)
     {
-        $logOutEndpoint = LoginSessionController::singleton()->Link('remove');
+        $logOutEndpoint = Director::baseURL() . LoginSessionController::singleton()->Link();
```

Comment:
This might not actually be needed. It was doing weird thing ... but that might have been my misconfigured route